File: jour_7/jour7.c
#include<stdio.h>
int a,b,c;
int main (void){
    printf("Entrez la premiere valeur : ");
    scanf("%d",&a);
    printf("Entrez la seconde valeur : ");
    scanf("%d",&b);
    printf("Entrez la troisieme valeur : ");
    scanf("%d",&c);
    if (a > b && a > c)
        printf("le plus grand est %d\n", a);
    else if (b > a && b > c)
        printf("le plus grand est %d\n", b);
    else if (c > a && c > b )
        printf("le plus grand est %d\n",c);
    else if (a == b && a == c)
        printf("%d %d %d sont egaux", a, b, c);
}

File: jour_7/jour7.pas
program TrouverLePlusGrand;

var
    a, b, c: integer;

begin
    write('Entrez la premiere valeur : ');
    readln(a);
    write('Entrez la seconde valeur : ');
    readln(b);
    write('Entrez la troisieme valeur : ');
    readln(c);

    if (a > b) and (a > c) then
    writeln('le plus grand est ', a)
    else if (b > a) and (b > c) then
    writeln('le plus grand est ', b)
    else if (c > a) and (c > b) then
    writeln('le plus grand est ', c)
    else if (a = b) and (a = c) then
    writeln(a, ' ', b, ' ', c, ' sont egaux');
    readln; 
end.